instrumentation/google.golang.org/grpc/otelgrpc/config.go

```diff
@@ -127,6 +128,20 @@ func (o meterProviderOption) apply(c *config) {
 	}
 }
 
+type includePeerAddrAsAttribute struct {
+	use bool
+}
+
+// WithPeerAddrAsAttribute returns an Option to use peer details when
+// recording with the meter rpc.server.duration.
+func WithPeerAddrAsAttribute(peerAddrAsAttribute bool) Option {
+	return includePeerAddrAsAttribute{use: peerAddrAsAttribute}
```

Comment:
I'm having doubts about this solution. The option seems extremely specific, and I wonder if we could find a more generic way of doing it.

Reply:
The meter here is specific to one type of interceptor. I could apply this to the spans as well if that's more preferable.